test/libsolidity/smtCheckerTests/external_calls/call_with_value_1.sol

```diff
@@ -2,13 +2,17 @@ contract C {
 	function g(address payable i) public {
 		require(address(this).balance == 100);
 		i.call{value: 10}("");
-		// Disabled due to Spacer nondeterminism
-		//assert(address(this).balance == 90); // should hold
-		// Disabled due to Spacer nondeterminism
-		//assert(address(this).balance == 100); // should fail
+		assert(address(this).balance == 90); // should hold
+		assert(address(this).balance == 100); // should fail
```

Comment:
While I do not disagree with this change. I wonder. Shouldn't we have a PR for one subject and another for something else? (bad wording maybe. jsut not having too much different things in one PR / commit).

Reply:
The code change is there exactly to allow re-enabling such tests.

Reply:
Yeah, the whole point was that these had to be disabled becuse they were failing on macOS so uncommenting them shows that the new setting works.

Reply:
I'll still enable more tests in a different PR, but wanted to enable at least one back in this one to prove the point and not bloat it.